pages/bancadas/[parliamentaryGroupSlug].js

```diff
@@ -38,14 +38,14 @@ export default function ParliamentaryGroup({
               congressperson_slug,
               id_name,
               id_second_surname,
-              link_photo,
+              plenary,
```

Comment:
Qué te parece esta manera @AlmeAld ?
```suggestion
              plenary: { link_photo },
```